Code/Sandbox/Plugins/CryDesigner/Objects/DesignerObject.cpp

```diff
@@ -684,6 +687,15 @@ void DesignerObjectFlags::Update()
 	ModifyFlag<uint64>(nFlags, ERF_FOB_ALLOW_DECAL_BLEND, !ignoreDecalBlend);
 	ModifyFlag<int>(statobjFlags, STATIC_OBJECT_NO_PLAYER_COLLIDE, excludeCollision);
 
+	if (disablePhysicsProxy && m_pObj->GetCompiler()->CheckFlags(ECompilerFlag::eCompiler_Physicalize))
```

Comment:
move "m_pObj->GetCompiler()->CheckFlags(ECompilerFlag::eCompiler_Physicalize)" out of the if for clarity